models/dataset.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Apr 14 20:36:19 2021
+
+@author: NLEGENDN
+"""
+import os
+import numpy as np
+import torch
+from torch.utils.data import DataLoader, random_split
+import functools
+import h5py
+from PIL import Image
+CUDA = torch.cuda.is_available()
+
+if CUDA:
+    CRYO_TRAIN_VAL_DIR = os.getcwd()+"/Cryo/VAE_Cryo_V3/Data/"
+else:
+    CRYO_TRAIN_VAL_DIR = os.getcwd()+"\\Data\\"
+
+
+KWARGS = {'num_workers': 1, 'pin_memory': True} if CUDA else {}
+
+
+def open_dataset(path, new_size, D3):
+    """
+    Parameters
+    ----------
+    path : string path.
+    new_size : int.
+    D3 : boolean if 2d or 3d
+
+    Returns
+    -------
+    tensor of images in black and white.
+
+    """
+    if not os.path.exists(path):
+        raise OSError
+    if path[-3:] == '.h5':
+        data_dict = h5py.File(path, 'r')
+        all_datasets = data_dict['particles'][:]
+    else:
+        all_datasets = np.load(path)
+    dataset = np.asarray(all_datasets)
+    img_shape = dataset.shape
+    N = img_shape[0]
```

Comment:
no uppercase if not a constant. if you use uppercase letters, python assumes it is either a constant variable or a Python class.

Also, `n_imgs` would be clearer as a name